<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qrec 1.0</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QSpinBox" name="spbFrames">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="lblSkipped">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblDebug">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="btnStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QSpinBox" name="spbScreen">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Buffersize [Mb]:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Frames</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="btnTest">
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLabel" name="lblBufferSize">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spbHeight">
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="value">
      <number>700</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="lblTime">
     <property name="text">
      <string>? [ms]</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="2">
    <widget class="QPushButton" name="btnExport">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="spbWidth">
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="value">
      <number>1400</number>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="btnStop">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="Line" name="line_2">
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="title">
     <property name="text">
      <string>Captured:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Screen</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Warnings:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="lblFrames">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Processed:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="lblProcessed">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
